Update interfaces/speech-api.idl

autofoolip opened the pull request:
This PR was automatically created by a bot.

Before merging, please check that any tests that depend on the updated IDL files still work.

If additional changes are needed, please manually create another PR based on this one.

See the [README](https://github.com/web-platform-tests/wpt/blob/master/interfaces/README.md) for how the IDL files in this directory are used.

<hr>

Source: https://github.com/tidoust/reffy-reports/blob/57753a9/whatwg/idl/speech-api.idl
Build: https://travis-ci.org/tidoust/reffy-reports/builds/435173976

foolip commented:
This PR looks like it's reverting changes in https://github.com/w3c/speech-api/pull/45 which haven't even be merged to the spec yet. That's because the changes were made in https://github.com/web-platform-tests/wpt/pull/13057. Rather than reverting the changes here, I'll see to getting the spec changes landed ASAP. Probably some whitespace differences will then remain in this PR.

autofoolip committed:
Update interfaces/speech-api.idl

Source: https://github.com/tidoust/reffy-reports/blob/57753a9/whatwg/idl/speech-api.idl
Build: https://travis-ci.org/tidoust/reffy-reports/builds/435173976

foolip reviewed (approved):
Alright, now the changes look sensible.
